lightfm/evaluation.py

```diff
@@ -35,9 +35,9 @@ def precision_at_k(
     test_interactions: np.float32 csr_matrix of shape [n_users, n_items]
          Non-zero entries representing known positives in the evaluation set.
     train_interactions: np.float32 csr_matrix of shape [n_users, n_items], optional
-         Non-zero entries representing known positives in the train set. These
```

Comment:
Since this is an evaluation routine, there is no training involved. Why do you say 'trained'?

Reply:
Aah, that's a mistake. Thanks, let me fix that.